geo-traits/src/point.rs

```diff
@@ -34,11 +34,11 @@ impl<T: CoordNum> PointTrait for Point<T> {
 }
 
 #[cfg(feature = "geo-types")]
-impl<T: CoordNum> PointTrait for &Point<T> {
-    type CoordType<'a>
+impl<'a, T: CoordNum> PointTrait for &'a Point<T> {
```

Comment:
I confess, I'm not great with lifetimes, but I feel like this is a no-op change?

Reply:
Well if we're all admitting that we're "not that great at lifetimes" I'll chime in and admit it too. I assume that this change is intended to make it clearer that the lifetime of the returned coordinate matches the lifetime of the source reference. But does "clearer" in this case actually mean "fixes a `borrowck` error"? Great if it does, and we'd love to see it – I thought the existing lifetime arrangement already made the relationship clear.